### ISSUE TITLE
Setting up aiy-python-wheels: protobuf not supported on armv6.1 revisited

### ISSUE THREAD

**Issue by genus1** [google/aiyprojects-raspbian#530]
This was reported in #525 but was closed by the reporter.
protobuf-3.6.1-cp35-cp35m-linux_armv6.1 is not a supported wheel on this platform.

dmitriykovalev https://github.com/dmitriykovalev marked this as a bug as stated below

"Thanks for reporting this. Technically this wheel is compatible and should work on RPi 3B+ but looks like pip is not happy about its name. Please use our SD card image if possible, it should work on any Pi board."

This is a challenge. I had five boards working before the upgrade to button.wait_for_press(). I have a LOT of stuff on the pi besides just voice to be able to control boards with voice. I literally have to start from scratch to rebuild these boards, which is really not a desired option. Please post an upgrade here when the bug is fixed using upgrade. Or in the meantime could you post the code to fix the bug you put in your new image? The problem is in duration_pb2.

https://community.mycroft.ai/t/setting-up-aiy-python-wheels-protobuf-not-supported-on-armv6-1/5130 reported a potential temporary solution but I have not checked if it worked yet.

The permanent solution is for find out why and fix protobuf on git so any updates or upgrades work.

**Comment by genus1** [comment]
I tried a new build as dmitriykovalev stated above from scratch version 2018-11-16.  Audio check did not work on pi3b+ and the voice V1 hat.  I added the dtoverlay=googlevoicehat-soundcard to config.txt and it still does not work.
I think the new build with raspberry adding video etc may be interfering with the update.

I did a new update and upgrade and it said all was up to date.  I rebooted and everything works fine.  It was just a pain to reconnect all my keys for voice etc at google, and I do not have the files created before on this system.  I will need to copy all the files down and back to the other and install software I was using.

Bottom line is that the 2018-11-16 upgrade works.  Now if Google will figure out what is causing the protobuf error and fix it for normal updates and upgrades this can be closed.

**Comment by manoj7410** [comment]
@genus1 I just tested build version 2018-11-16 on my RPI 3B+ and check_audio is working fine on it. I have downloaded the build from https://github.com/google/aiyprojects-raspbian/releases/tag/v20181116

**Comment by genus1** [comment]
New version works.

**Comment by phaniteja1** [comment]
I am still getting this error. I did sudo update and everything but I still get an error message saying that protobuf is not supported on my raspberrypi3. I followed the instructions in HACKING.md. Please help!

**Comment by mss-deepak** [comment]
> @genus1 I just tested build version 2018-11-16 on my RPI 3B+ and check_audio is working fine on it. I have downloaded the build from https://github.com/google/aiyprojects-raspbian/releases/tag/v20181116

is this bookable image? as I am trying to add this on my card and its not booting.
